src/bundle.c

```diff
@@ -413,7 +410,7 @@ int install_bundles(struct list *bundles, int current_version, struct manifest *
 
 	if (ret) {
 		if (ret == 1) {
-			printf("There are no pending bundles to install, exiting now\n");
+			printf("%s bundle already installed, exiting now\n", bundle);
```

Comment:
Want to know how I know I didn't run make check... I clearly was in the middle of something else with this change X(.